pkg/controller/deployment/sync.go

```diff
@@ -109,7 +109,7 @@ func (dc *DeploymentController) getAllReplicaSetsAndSyncRevision(deployment *ext
 	// List the deployment's RSes & Pods and apply pod-template-hash info to deployment's adopted RSes/Pods
 	rsList, podList, err := dc.rsAndPodsWithHashKeySynced(deployment)
 	if err != nil {
-		return nil, nil, fmt.Errorf("error labeling replica sets and pods with pod-template-hash: %v", err)
+		return nil, nil, fmt.Errorf("error labeling ReplicaSets and pods with pod-template-hash: %v", err)
```

Comment:
@krmayankk if you wish go ahead and switch "pods" to "Pods", rebase, squash, and I'll take it. Just don't change anything in the events the controller emits, because that will break test upgrades from older versions.